pages/_app.js

```diff
@@ -1,4 +1,4 @@
-import "tailwindcss/tailwind.css";
```

Comment:
I'm not sure what you were doing here. Following the blog article I sent, it says to give tailwind its own css file([source](https://medium.com/better-programming/how-to-set-up-next-js-with-tailwind-css-b93ccd2d4164#366e)).